sdk/mixedreality/mixed-reality-authentication/swagger/README.md

```diff
@@ -21,6 +21,7 @@ use-extension:
 hide-clients: true
 package-version: 1.0.0-beta.1
 v3: true
+use-core-v2: true
```

Comment:
Isnt this the default option?

Reply:
You are right, that is the default. I am going to remove that line.